module/utils/databases/Database.mjs

```diff
@@ -289,7 +293,28 @@ export class Database {
 	 * @param {Notification} notif The progress bar notification used for
 	 * user feedback while performing migrations.
 	 */
-	static async migrateData(lastVersion, notif) {};
+	static async migrateData(lastVersion, notif) {
+		const totalSteps = 1;
+		Logger.debug(lastVersion);
+
+		/*
+		This migration is for going up to 1.0.3, getting rid of any tables that have
+		a bucket type of range, since those were not supported within the initial
+		release, but could still accidentally be created by users.
+		*/
+		if (isNewerVersion(`1.0.3`, lastVersion)) {
```

Comment:
I'm not entirely sold on this, as it makes the (relatively) heavy-handed assumption that plugin version is directly correlated with the data format, which may not be the case for some database adapters, and I feel... iffy about it.

It's probably fine for now though, so I don't think it's particularly worth over-engineering until there's a case where data format isn't directly correlated to plugin version.